component.yaml

```
@@ -18,7 +18,7 @@ inputs:
   - key: "AC_RN_TEST_COMMAND_ARGS"
     isRequired: false
     title: Extra commands
-    description: Specify additional command arguments for running Jest tests, such as `--debug`. For more information, see the Jest [CLI options](https://jestjs.io/docs/cli#options).
+    description: Specify additional command arguments for running Jest tests. An extra parameter will be added to the end of the command `jest --coverage --coverageDirectory=coverage --coverageReporters=lcov` which will be used by default. For more information, see the Jest [CLI options](https://jestjs.io/docs/cli#options).
```

I'm trying to add an example because I get a lot of questions from customers about how to use this.
```suggestion
    description: Specify additional command arguments for running Jest tests. An extra parameter will be added to the end of the command `jest --coverage --coverageDirectory=coverage --coverageReporters=lcov` which will be used by default. You can add extra arguments, such as `--debug --colors`, without affecting the default ones. For more information, see the Jest [CLI options](https://jestjs.io/docs/cli#options).
```